src/Microsoft.DotNet.Darc/src/Darc/Properties/launchSettings.json

```diff
@@ -2,8 +2,8 @@
   "profiles": {
     "Microsoft.DotNet.Darc": {
       "commandName": "Project",
-      "commandLineArgs": "get-dependency-graph --repo https://github.com/dotnet/core-sdk -v 3fa1e2c5cfa5142c7b9c4f10ab1d0d74a8111f34",
-      "workingDirectory": "G:\\Code\\jcagme\\arcade-services"
+      "commandLineArgs": "get-subscriptions --batchable",
+      "workingDirectory": "D:\\enlistments\\core-sdk"
```

Comment:
I'll add this to .gitignore (eventually :))

Reply:
I haven't been able to get this thing to actually .gitignore.  Not sure why.